### DISCUSSION TITLE
Website announcement!

### DISCUSSION THREAD

**Discussion by LouisGariepy** [cornucopia-rs/cornucopia#51] [Announcements]
I'm pleased to announce I've been in touch with someone willing to contribute a Svelte website for the project. I'm not a frontend dev myself, but I think this is an exciting possibility. I've already seen some early iterations and I'm super pleased.

So, this post acts as an announcement, but I'd also like to use it as a platform to collect suggestions for the website. If you have an idea (anything really, aesthetic, technical, or otherwise), please share it here!

**Comment by Virgiel** [comment]
Any news on this?

**Comment by LouisGariepy** [comment]
I've been presented with a fully working sketch for the website yesterday! They are using svelte+tailwind, and the site would be hosted on Netlify.

I'm having a hard time giving feedback to them because I'm not much of a webdev/designer. I'll open an empty repo in our org for the website, and let them know to open a fork so we can start giving feedback and iterating.

The other problem is that we are having trouble finding a good way to highlight code snippets on the site.
